### ISSUE TITLE
Design screenshots followup

### ISSUE THREAD

**Issue by lsl** [Automattic/wp-calypso#49097]
Following on from #43428 there are a few things outstanding before we can launch the mshots implementation.

* [x] The vertically oriented screenshots aren't _quite_ how we'd like them, some of the viewport height based elements appear stretched vertically a little bit more than desired. This may or may not be a deal-breaker. This could be achieved by setting custom viewport sizes per design in available-designs-config.json. ([Thanks](https://github.com/Automattic/wp-calypso/pull/43428#issuecomment-650921273) @deBhal) - switching to landscape #49170
* [x] Currently the image file size is a bit too much greater than the existing webp approach. Separately to this PR we're [proposing adding in a quality param](https://github.com/Automattic/mShots/pull/23) for mShots to support smaller file sizes. Calypso only suggestion [here.](https://github.com/Automattic/wp-calypso/pull/43428#discussion_r560737507) - reducing image HxW params fixes this #49170
* [x] The logic to keep the image hidden until _really_ loaded is a bit hacky and relies on a hard-coded URL. Is this safe enough, and does it hold up to testing when generating new screenshots? Is there a reliable way to test this, or write a test for it? - tests using image heights, has caveats but seems to work well enough.
* [x] Are we caching correctly? If a user goes back and forward between steps, does the browser re-download the images, or does it correctly cache them? We should ensure that the images aren't being re-downloaded multiple times. - mshots images get cached
* [x] Implement max reload attempt counts https://github.com/Automattic/wp-calypso/pull/43428#discussion_r444082159 done in #43428
* [x] Look at [loading images in a non-visible location](https://github.com/Automattic/wp-calypso/pull/43428#issuecomment-763408492) in earlier steps then moving them in later without re-creating the html image elements. This should work in theory but may be too difficult to work into current step architecture. Suggest: mess with absolute positioning off screen.
Related to #37665
* [x] Change design layout to incorporate different height designs: https://github.com/Automattic/wp-calypso/pull/48727 See: #49170 and discussion in #48727
* [ ] Handle 404 gracefully - less of an issue for the previews, more of an issue for the MShotsImage component for reuse.
* [ ] Clean up sizing of images / breakpoints https://github.com/Automattic/wp-calypso/pull/49170#issuecomment-767476316 tracked: https://github.com/Automattic/wp-calypso/issues/49382
* [ ] Merge [mShots Docker setup](https://github.com/Automattic/mShots/pull/22)
* [ ] Merge [mShots quality option](https://github.com/Automattic/mShots/pull/23)

**Comment by simison** [comment]
> Clean up sizing of images / breakpoints

@lsl this would prolly be good to track as its own issue since it's more about UI and less about the infra-work in mshots.

**Comment by lsl** [comment]
#49382 is tracking the UI change moving forward and I've moved the mShots issues into our board backlog.
